### PR TITLE
MAT-6332 verifying dependencies and dev dependencies

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-6332](https://jira.cms.gov/browse/MAT-6332)
(Optional) Related Tickets:

### Summary
verified dependencies and dev dependencies

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +49,13 @@
     "webpack-pwa-manifest": "^4.3.0"
   },
   "dependencies": {
-    "single-spa": "^5.9.4"
+    "@madie/madie-auth": "^0.0.2",
+    "@madie/madie-editor": "^0.0.2",
+    "@madie/madie-layout": "^0.0.2",
+    "@madie/madie-measure": "^0.0.2",
```
Comment (RohitKandimalla) [inline package.json:55]: I don't think we need these dependencies

**Review by RohitKandimalla — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sb-prateekkeerthi) [reply package.json:55]: but these dependencies are being used in webpack.config.js file
